notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>If you are in a live tutorial session, you might be using a shared test account or project. To avoid name collisions between users on the resources created, <strong><s>you</s></strong> create a timestamp for each instance session, and append the timestamp onto the name of resources you create in this tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Done.</p>